pkg/sql/importer/import_job.go

```diff
@@ -1394,6 +1394,7 @@ func (r *importResumer) OnFailOrCancel(ctx context.Context, execCtx interface{},
 		ctx context.Context, txn *kv.Txn, descsCol *descs.Collection,
 	) error {
 		if err := r.dropTables(ctx, txn, descsCol, cfg); err != nil {
+			log.Errorf(ctx, "drop tables failed: %s", err.Error())
```

Comment:
Interesting, just out of curiosity, where does this error get eaten currently that we don't see it _somewhere_ as the result of returning it below?  I have a similar question about the other log statements.

Reply:
I believe somewhere in the retry logic of `DescTxnWithExecutor()` https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/internal.go#L1414

This func suggests to me that retries only occur if the two version descriptor invariant was violated, which seems independent of what we've hypothesized. Hopefully these log lines will shed some light on this!